src/components/transactions/GovDelegation/GovDelegationModalContent.tsx

```diff
@@ -97,42 +119,83 @@ export const GovDelegationModalContent = () => {
-        <Trans>Asset to delegate</Trans>
+      <Typography variant="description" color="text.secondary" sx={{ mb: 1 }}>
+        <Trans>{isRevokeModal ? 'Power to revoke' : 'Power to delegate'}</Trans>
       </Typography>
+      <DelegationTypeSelector
+        disableVoting={
+          isRevokeModal &&
+          !!powers &&
+          powers.aaveVotingDelegatee === '' &&
+          powers.stkAaveVotingDelegatee === ''
+        }
+        disableProposing={
+          isRevokeModal &&
+          !!powers &&
+          powers.aavePropositionDelegatee === '' &&
+          powers.stkAavePropositionDelegatee === ''
+        }
+        delegationType={delegationType}
+        setDelegationType={setDelegationType}
+      />
+
+      {isRevokeModal ? (
+        <Typography variant="description" color="text.secondary" sx={{ mt: 6, mb: 2 }}>
+          <Trans>Balance to revoke</Trans>
+        </Typography>
+      ) : (
+        <TextWithTooltip
+          text="Balance to delegate"
+          variant="description"
+          textColor="text.secondary"
+          wrapperProps={{ mt: 6, mb: 2 }}
+        >
+          <Trans>
+            Choose how much voting/proposition power to give to someone else by delegating some of
+            your AAVE or stkAAVE balance. Your tokens will remain in your account, but your delegate
+            will be able to vote or propose on your behalf. If your AAVE or stkAAVE balance changes,
+            your delegate&apos;s voting/proposition power will be automatically adjusted.
+          </Trans>
+        </TextWithTooltip>
+      )}
+
       <DelegationTokenSelector
         setDelegationToken={setDelegationToken}
         delegationTokenAddress={delegationToken}
         delegationTokens={tokens}
-      />
-      <Typography variant="description">
-        <Trans>Type of delegation</Trans>
-      </Typography>
-      <DelegationTypeSelector
         delegationType={delegationType}
-        setDelegationType={setDelegationType}
+        filter={isRevokeModal}
       />
-      <Typography variant="description">
-        <Trans>Recipient address</Trans>
-      </Typography>
-      <FormControl error={delegateAddressBlockingError !== undefined} variant="standard" fullWidth>
-        <TextField
-          variant="outlined"
-          fullWidth
-          value={delegate}
-          onChange={(e) => setDelegate(e.target.value)}
-          placeholder={t`Enter ETH address`}
-          error={delegateAddressBlockingError !== undefined}
-          helperText={handleDelegateAddressError()}
-        />
-      </FormControl>
+      {!isRevokeModal && (
+        <>
+          <Typography variant="description" color="text.secondary" mb={1}>
+            <Trans>Recipient address</Trans>
+          </Typography>
+          <FormControl
+            error={delegateAddressBlockingError !== undefined}
+            variant="standard"
+            fullWidth
+          >
+            <TextField
```

Comment:
We should look at making a component for the input that looks up an address from an ENS name or UD (for watch mode) and use that here.